src/Map/Map.cpp

```diff
@@ -0,0 +1,200 @@
+#include "Map.h"
+#include <iostream>
+#include <list>
+
+namespace Map {
+
+    Country::Country(const string &name) {
+        this->name = new string(name);
+        this->index = new int(-1);
+        this->continent_index = new int(-1);
+        this->nb_armies = new int(0);
```

Comment:
what's the difference between init -1 or 0

Reply:
0 would mean index 0... -1 means no index set yet